src/de/dkfz/b080/co/aceseq/ACESeqWorkflow.java

```diff
@@ -76,20 +85,21 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
 
 
         Tuple2<TextFile, TextFile> breakpoints = ACESeqMethods.pscbsGaps(haplotypedSNPFile, correctedWindowFile.value0, annotationResult.getGenderFile());
-        Tuple2<TextFile, TextFile> mergedSvs = null;
+        Tuple2<SVFile, TextFile> mergedSvs = null;
 
-        if (runWithSv) {
-            mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSv); // true is passed
+        if (runWithSV) {
+            mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSV); // true is passed
+            if (mergedSvs == null) {
+                return allowMissingSVFile(context); // It might be allowed to exit here without an error
```

Comment:
Why "might"? Is it or not? Are you unsure? Is this comment a question?

Reply:
Depends in the value of the allow variable... So is might a wrong word?